content/SCALE/SCALEUIReference/DataProtection/VMwareSnapshotsScreen.md

```diff
@@ -9,20 +9,30 @@ tags:
 - vm
 ---
 
-Use the **VMware Snapshot Integration** option on the **Data Protection > Periodic Snapshot Tasks** widget to create snapshots when you are using TrueNAS SCALE as a VMWare datastore. 
-Click  **VMware Snapshot Integration** to display the **Add VMware Snapshot** screen.
+Use the **VMware Snapshot Integration** option on the **Data Protection > Periodic Snapshot Tasks** widget to create snapshots when you are using TrueNAS SCALE as a VMWare datastore.
+See [Creating VMWare Snapshots]({{< relref "creatingvmwaresnapshots.md" >}}) for a detailed tutorial.
 
-   {{< trueimage src="/images/SCALE/DataProtection/emptyvmwaresnapshotadd.png" alt="Add VMware Snapshot Screen" id="Add VMware Snapshot Screen" >}}
+Click **VMware Snapshot Integration** to open the **VMWare Snapshots** screen.
+
+{{< trueimage src="/images/SCALE/DataProtection/vmwaresnapshottask.png" alt="VMware Snapshot Integration" id="VMware Snapshot Integration" >}}
+
+Click the **Add** button to display the **Add VMware Snapshot** screen.
```

Comment:
This is an older method of documenting the ui functions, suggest making these changes to update it to what we are currently doing.
```suggestion
**VMware Snapshot Integration**  opens the **VMWare Snapshots** screen.

{{< trueimage src="/images/SCALE/DataProtection/vmwaresnapshottask.png" alt="VMware Snapshot Integration" id="VMware Snapshot Integration" >}}

**Add** opens the **Add VMware Snapshot** screen.
```